requirements.txt

```diff
@@ -1,7 +1,5 @@
 pyodbc
 six
-sqlalchemy
```

Comment:
Why is `sqlalchemy` being removed here? Is it being handled as a requirement/dependency somewhere else?